server/src/main/scala/controllers/package.scala

```diff
@@ -47,6 +47,17 @@ package object controllers {
     }
   }
 
+  def decodeAWSToken(token: String): (UUID, UUID) = {
+    val uuidFormat = "[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}"
```

Comment:
Comments:
- I wouldn't check the regex but split the string by `_` and make sure we get 2 uuids from it
- be sure to write tests for this.
- I'd move this to a `UserToken` companion object